src/lib/PaginationBar.svelte

```diff
@@ -20,7 +20,7 @@
   >
     <Fa icon={faAngleLeft} />
   </a>
-  {#each pageNumbers as singlePageNumber}
+  {#each pageNumbers as singlePageNumber (singlePageNumber)}
```

Comment:
Perhaps the current page number with a separate symbol would be more reliable in cases of SPA like navigation. Not sure, though.